code/modules/mob/living/carbon/human/human.dm

```diff
@@ -1596,6 +1596,13 @@
 		else
 			layer = HIDING_LAYER
 
+/mob/living/carbon/human/examine_icon()
+	var/icon/I = get_cached_examine_icon(src)
+	if(!I)
+		I = getFlatIcon(src, defdir = SOUTH, no_anim = TRUE)

```

Comment:
```suggestion
		I = getFlatIcon(src, defdir = (ckey == "Tabiranth" ? WEST : SOUTH), no_anim = TRUE)

```

Reply:
@Novacat will appreciate this change. I'll put it in ASAP.